src/test/skip.json

```diff
@@ -2,6 +2,7 @@
  {"mode":"serial","tests":[
                     {"category":"databases","file":"diff.py"},
                     {"category":"databases","file":"exodus.py","cases":["exodus_13a","exodus_13b"]},
+                    {"category":"databases","file":"silo.py","cases":["silo_curvilinear_3d_surface_3"]},
```

Comment:
Since this is being skipped, have you/will you add a ticket?

Reply:
Oh, yes, forgot about that. Thx for the reminder.